src/Microsoft.Azure.WebJobs.Host/Blobs/Triggers/BlobTriggerAttributeBindingProvider.cs

```diff
@@ -45,7 +46,8 @@ internal class BlobTriggerAttributeBindingProvider : ITriggerBindingProvider
             IContextSetter<IMessageEnqueuedWatcher> messageEnqueuedWatcherSetter,
             ISharedContextProvider sharedContextProvider,
             SingletonManager singletonManager,
-            TraceWriter trace)
+            TraceWriter trace,
+            string hostMachineId)
```

Comment:
Good, I was just about to comment that you update all the param names for clarity :)